packages/ui/src/utils/stringUtils.ts

```diff
@@ -0,0 +1,4 @@
+export const camelcasetoString = (proxy: string) =>
```

Comment:
can you change the name of the function and file to be `camelcaseToString` and `camelcaseToString.ts` and add it to the index.ts here https://github.com/ChainSafe/Multix/blob/1a82bd8cbc2ffefed5bb087ee8cf32afd54c4c1f/packages/ui/src/utils/index.ts ? The name of the file was too generic IMHO.

Reply:
It's definitely too generic. If another function comes related to string manupulation though, it would be cleaner to put them in a single stringUtils file.

I renamed the file in a new commit #[840f3eb](https://github.com/ChainSafe/Multix/pull/490/commits/840f3ebef7bf61e4c3df0dd30e06f49b59b70d78)